content/en/docs/best-practices/skywalking-solution-for-openfunction.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Use SkyWalking for OpenFunction as an Observability Solution"
+linkTitle: "Use SkyWalking for OpenFunction as an Observability Solution"
+weight: 5300
+description: >	
+    Learn how to use SkyWalking for OpenFunction as an observability solution.
+---
+
+This document describes how to use SkyWalking for OpenFunction as an observability solution.
+
+## Overview
+
+Although FaaS allows developers to focus on their business code without worrying about the underlying implementations, it is difficult to troubleshoot the service system. OpenFunction tries to introduce capabilities of observability to improve its usability and stability.
+
+SkyWalking provides solutions for observing and monitoring distributed systems in many different scenarios. OpenFunction has bundled go2sky(SkyWalking's Golang agent) in OpenFunction tracer options to provide distributed tracing, statistics of function performance, and functions dependency map.
+
+## Prerequisites
+
+- You have [installed OpenFunction](../../getting-started/installation/).
+- You have [created a secret](../../getting-started/your-first-function/function-go/#create-a-secret).
+- You have [installed SkyWalking](https://github.com/apache/skywalking-kubernetes#apache-skywalking-kubernetes).
```

Comment:
It should be updated to `installed SkyWalking v9` since the FaaS support has just been introduced in SkyWalking v9, see [this comment](https://github.com/apache/skywalking/pull/8789#discussion_r841063853) for the context.

Reply:
[Updated](https://github.com/OpenFunction/website/pull/54/commits/1cf62c9e09b0cf4e62de025a5c4ad5d48f74dd30).